lib/src/screens/home/feedTab.dart

```diff
@@ -12,8 +19,144 @@ class FeedTab extends StatefulWidget {
 }
 
 class _FeedTabState extends State<FeedTab> {
+  static final DateFormat postDateFormatter = DateFormat('h:m a');
+
+  final List<FeedItem> feedItems = [
+    FeedItem(
+      name: 'Lecture 1',
+      owner: 'Calculus I - AAB',
+      body: 'Please do this IMMEDIATELY',
+      type: FeedItemType.Lecture,
+      itemCode: 'CS_124_74481',
+      postDate: DateTime.now().add(Duration(hours: 4)),
+      attachmentUrl:
+          'https://images.all-free-download.com/footage_preview/mp4/kitty_68.mp4',
+    ),
+    FeedItem(
+      name: 'Homework 5',
+      owner: 'English 101',
+      body: 'Never do this',
+      type: FeedItemType.Homework,
+      itemCode: 'CS_124_74482',
+      postDate: DateTime.now().add(Duration(hours: 2)),
+    ),
+  ];
+
   @override
   Widget build(BuildContext context) {
-    return Text("Base");
+    final size = MediaQuery.of(context).size;
+
+    return Container(
+      width: size.width,
+      height: size.height,
+      child: ListView.builder(
+        itemCount: feedItems.length,
+        itemBuilder: (context, i) => _buildItem(context, feedItems[i]),
+      ),
+    );
+  }
+
+  Widget _buildItem(BuildContext context, FeedItem item) {
+    return Padding(
+      padding: EdgeInsets.all(10),
+      child: Card(
+        color: MediaQuery.of(context).platformBrightness == Brightness.dark
+            ? Colors.grey[900]
+            : Colors.grey[300],
+        child: InkWell(
+          onTap: () => Navigator.of(context).push(
+            platformPageRoute(
+              context: context,
+              builder: (context) {
+                return _getScreenForFeedItem(item);
+              },
+            ),
+          ),
```

Comment:
```suggestion
          onTap: () => Navigator.of(context, rootNavigator: true).push(
            CupertinoPageRoute(
              builder: (context) {
                return _getScreenForFeedItem(item);
              },
            ),
          ),
```
I know I've been pushing for using multiplatform stuff, but for page routes, I think it's better to use CupertinoPageRoute, they feel more intuitive to use than MaterialPageRoute from a user's perspective. Let me know what you think.